pkgs/top-level/ocaml-packages.nix

```diff
@@ -543,6 +543,10 @@ let
 
     jsonm = callPackage ../development/ocaml-modules/jsonm { };
 
+    junit = callPackage ../development/ocaml-modules/junit { };
+    junit_ounit = callPackage ../development/ocaml-modules/junit/junit_ounit.nix { };
+    junit_alcotest = callPackage ../development/ocaml-modules/junit/junit_alcotest.nix { };
+
```

Comment:
This is just a matter of taste, but you don’t have to repeat the `junit_` prefix in the file names (it is already the name of the parent directory).

Reply:
Done.